examples/weather/pangu_weather/train_pangu_lite_era5.py

```diff
@@ -0,0 +1,331 @@
+            + torch.sum(torch.pow(predvar_upper_air - outvar_upper_air, 2))
+        ) / (num_elements_surface + num_elements_upper_air)
+
+        num_examples += predvar_surface.shape[0]
+
+    pangu_model.train()
+    return loss_epoch / num_examples
+
+
+@hydra.main(version_base="1.2", config_path="conf", config_name="config")
+def main(cfg: DictConfig) -> None:
+    DistributedManager.initialize()
+    dist = DistributedManager()
+
+    # Initialize loggers
+    # initialize_wandb(
+    #     project="Modulus-Launch-Dev",
+    #     entity="Modulus",
+    #     name="FourCastNet-Training",
+    #     group="FCN-DDP-Group",
+    # )
+    initialize_mlflow(
+        experiment_name="Modulus-Launch-Dev",
+        experiment_desc="Modulus launch development",
+        run_name="Pangu-Training",
+        run_desc="Pangu ERA5 Training",
+        user_name="Modulus User",
+        mode="offline",
+    )
+    LaunchLogger.initialize(use_mlflow=cfg.use_mlflow)  # Modulus launch logger
+    logger = PythonLogger("main")  # General python logger
+
+    no_channals_pangu = 4 + 5 * 13
+    datapipe = ERA5HDF5Datapipe(
+        data_dir="/data/train/",
+        stats_dir="/data/stats/",
+        channels=[i for i in range(no_channals_pangu)],
+        num_samples_per_year=cfg.num_samples_per_year_train,
+        batch_size=1,
+        patch_size=(1, 1),
+        num_workers=8,
+        device=dist.device,
+        process_rank=dist.rank,
+        world_size=dist.world_size,
+    )
+    logger.success(f"Loaded datapipe of size {len(datapipe)}")
+
+    mask_dir = "/data/constant_mask"
+    land_mask = torch.from_numpy(
+        np.load(os.path.join(mask_dir, "land_mask.npy")).astype(np.float32)
+    )
+    soil_type = torch.from_numpy(
+        np.load(os.path.join(mask_dir, "soil_type.npy")).astype(np.float32)
+    )
+    topography = torch.from_numpy(
+        np.load(os.path.join(mask_dir, "topography.npy")).astype(np.float32)
+    )
+    surface_mask = torch.stack([land_mask, soil_type, topography], dim=0).to(
+        dist.device
+    )
+    logger.success(f"Loaded suface constant mask from {mask_dir}")
+
+    if dist.rank == 0:
+        logger.file_logging()
+        validation_datapipe = ERA5HDF5Datapipe(
+            data_dir="/data/test/",
+            stats_dir="/data/stats/",
+            channels=[i for i in range(no_channals_pangu)],
+            num_steps=1,
+            num_samples_per_year=4,
+            batch_size=1,
+            patch_size=(1, 1),
+            device=dist.device,
+            num_workers=8,
+            shuffle=False,
+        )
+        logger.success(f"Loaded validaton datapipe of size {len(validation_datapipe)}")
+
+    pangu_model = Pangu(
+        img_size=(721, 1440),
+        patch_size=(2, 8, 8),
+        embed_dim=192,
+        num_heads=(6, 12, 12, 6),
+        window_size=(2, 6, 12),
+    ).to(dist.device)
+
+    if dist.rank == 0 and wandb.run is not None:
+        wandb.watch(
```

Comment:
I might be wrong but seems like we are not using wandb anymore or at least in this script. Seems like its using mlflow now?